clubs.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Clubs
+priority: 4
+description: The UUU clubs directory
+image: assets/images/uniclub.jpg
+nav-menu: true
+menu_bottom: "test"
+show_tile: true
+---
```

Comment:
You dropped the google map showing the locations. Please add this back.

Reply:
I dropped the map because it sticks out relative to the rest of the design. It's a good tool and I would like to add it back in the future, but I first have to bring the layout to a mature point so that I can put it somewhere sensible. For now, I think the list of clubs will do.